packages/math/src/Matrix.js

```diff
@@ -361,6 +361,7 @@ export default class Matrix
         const skewX = -Math.atan2(-c, d);
         const skewY = Math.atan2(b, a);
 
+        const rad2deg = transform.useRadians ? 1.0 : (180.0 / Math.PI);
```

Comment:
Consider `import { RAD_TO_DEG } from './const'` and using `RAD_TO_DEG` here.

Reply:
That'll be extra import line, I think my version is more verbose.

Reply:
With rollup it’s all flattened anyways so the extra import doesn’t matter. I’m fact, it’s better to import because this uses a single instance variable and compressed smaller. Verbosity is not useful here and constants  should be declared, imo.